CHANGELOG.md

```diff
@@ -2,6 +2,142 @@
+- Fix heuristic object properties being broken in some cases / fix DepthMeter being ignored sometimes [#4679](https://github.com/rerun-io/rerun/pull/4679)
+
+#### 🌁 Viewer Improvements
+- Make viewer contexts's render context reference non-mutable [#4430](https://github.com/rerun-io/rerun/pull/4430)
+- Pass viewer context always non-mutable [#4438](https://github.com/rerun-io/rerun/pull/4438)
+- The Rerun Viewer can now consume from stdin
+  - Standard input/output support 1: stream RRD data from stdin [#4511](https://github.com/rerun-io/rerun/pull/4511)
+  - Standard input/output support 2: Rust SDK stdout impl/examples/docs [#4512](https://github.com/rerun-io/rerun/pull/4512)
+  - Standard input/output support 3: Python SDK stdout impl/examples/docs [#4513](https://github.com/rerun-io/rerun/pull/4513)
+  - Standard input/output support 4: C++ SDK stdout impl/examples/docs [#4514](https://github.com/rerun-io/rerun/pull/4514)
+- Support for custom DataLoaders:
+  - `DataLoader`s 0: utility for hierarchical `EntityPath` from file path [#4516](https://github.com/rerun-io/rerun/pull/4516)
+  - `DataLoader`s 1: introduce, and migrate to, `DataLoader`s [#4517](https://github.com/rerun-io/rerun/pull/4517)
+  - `DataLoader`s 2: add text-based `DataLoader` (`.txt`, `.md`) [#4518](https://github.com/rerun-io/rerun/pull/4518)
+  - `DataLoader`s 3: add 3D point cloud `DataLoader` (`.ply`) [#4519](https://github.com/rerun-io/rerun/pull/4519)
+  - `DataLoader`s 4: add generic folder `DataLoader` [#4520](https://github.com/rerun-io/rerun/pull/4520)
+  - `DataLoader`s 5: add support for external binary `DataLoader`s (PATH) [#4521](https://github.com/rerun-io/rerun/pull/4521)
+  - `DataLoader`s 6: first-class support for `Incompatible` [#4565](https://github.com/rerun-io/rerun/pull/4565)
+  - `DataLoader`s 7: support for custom `DataLoader`s [#4566](https://github.com/rerun-io/rerun/pull/4566)
+- 3D->2D & 2D->3D selection visualizations stick now around on selection [#4587](https://github.com/rerun-io/rerun/pull/4587)
+- Fix incorrect bounding box calculation for camera view parts [#4640](https://github.com/rerun-io/rerun/pull/4640)
+
+#### 🚀 Performance Improvements
+- Parallelize Space View system evaluation [#4460](https://github.com/rerun-io/rerun/pull/4460)
+- Limit server memory [#4636](https://github.com/rerun-io/rerun/pull/4636)
+
+#### 🧑‍🏫 Examples
+- Add nuScenes-based lidar examples [#4407](https://github.com/rerun-io/rerun/pull/4407) (thanks [@roym899](https://github.com/roym899)!)
+- Nightly builds [#4505](https://github.com/rerun-io/rerun/pull/4505)
+- Add LLM token classification example [#4541](https://github.com/rerun-io/rerun/pull/4541) (thanks [@roym899](https://github.com/roym899)!)
+- Use TOML in examples [#4553](https://github.com/rerun-io/rerun/pull/4553)
+
+#### 📚 Docs
+- Shared recordings 3: add how-to guide [#4385](https://github.com/rerun-io/rerun/pull/4385)
```

Comment:
this should be on the highlights - afaik previously one could not log to the same recording from several processes